react/__mocks__/vtex.dreamstore-icons.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+export const IconClose = ({ orientation, size, className }) => (
```

Comment:
Since both mocks are the same, it's nice to have a generic function and use it in `IconClose` and `IconSearch`.
Also, add some classnames do differ them, like `icon-close-mock`